server/src/controllers/users/user.js

```diff
@@ -62,6 +62,7 @@ exports.get = (req, res) => {
  */
 exports.create = (req, res) => {
   req.log.debug("POST/create");
+  console.log('Request', req);
```

Comment:
Please don't do this.  Use `.env` and change the `LOG_LEVEL` to debug you will see the whole request and response payload on your console.  I won't accept this kind of changes on our main repo.

Reply:
I remove this console log in the next commit